docker/TomcatContextUtil.pm

```diff
@@ -61,6 +61,58 @@ The default value to return. Omit to treat this secret as non-optional.
 
 =back
 
+=head1 get_resource
+
+Return the data for a JDBC resource.
+Fetch data using the supplied method (e.g. reading environment vars or files).
+
```

Comment:
Explain why this function exists - to use OpenShift/Kubernetes Secrets/Config Maps